src/rewrite_app_repos.sh

```diff
@@ -62,21 +83,7 @@ do
         # This is not ideal, however there was no better alternative as packagist only works with files in a github repo.
         echo "Building assets for ${reponame} at branch ${branch}"
 
-        git clone --recurse-submodules --single-branch --branch "${branch}" https://github.com/greenpeace/"${reponame}"
-
-        time npm ci --prefix "${reponame}" "${reponame}"
-        time npm run-script --prefix "${reponame}" build
-
-        if [[ "${reponame}" == *theme ]]; then \
-            subdir="themes"; \
-        else \
-            subdir="plugins"; \
-        fi; \
-
-        buildDir="${built_assets_dir}/public/wp-content/${subdir}/${reponame}/assets/build/"
-        mkdir -p "${buildDir}"
-        cp -a "${reponame}/assets/build/." "${buildDir}"
-        rm -rf "${reponame}"
+        build_assets "$branch" "$reponame" &
```

Comment:
Maybe we can time each of these tasks, so we know which repo is slowest and should be optimized first?